packages/vite/src/node/__tests__/utils.spec.ts

```diff
@@ -270,3 +272,15 @@ describe('isFileReadable', () => {
     })
   }
 })
+
+describe('processSrcSetSync', () => {
+  test('replace srcset with base url /base', async () => {
+    const devBase = '/base'
```

Comment:
```suggestion
    const devBase = '/base/'
```

The resolved `base` would always have a trailing slash, so it might be a good idea to do so to be consistent.